### PR TITLE
use consistent naming for WebVRConfig boolean options (fixes #100)

### LINKED ISSUE #100: Make WebVRConfig options consistent
IMO, all boolean names should be changed to be truthy ones. We already have `FORCE_ENABLE_VR`, `ENABLE_DEPRECATED_API`, `DEFER_INITIALIZATION`, `MOUSE_KEYBOARD_CONTROLS_DISABLED`. Let's make the naming consistent.

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -15,34 +15,76 @@
 var Util = require('./util.js');
 var WebVRPolyfill = require('./webvr-polyfill.js');
 
+var RENAMED_BOOL_CONFIG_OPTIONS = {
```
Comment (borismus) [inline src/main.js]: TBH, I'd err on simplicity and just rename the flags without back compat. It's so early for WebVR, I think it's ok. Not that many people use this polyfill yet. Even fewer use the WebVRConfig options at all. And those that do shouldn't expect updates to not have somewhat breaking changes.

File: src/main.js [hunk 2]
```diff
@@ -15,34 +15,76 @@
...
-  MOUSE_KEYBOARD_CONTROLS_DISABLED: false,
+  MOUSE_KEYBOARD_CONTROLS: true,
 
   // Prevent the polyfill from initializing immediately. Requires the app
   // to call InitializeWebVRPolyfill() before it can be used.
   DEFER_INITIALIZATION: false,
 
   // Enable the deprecated version of the API (navigator.getVRDevices).
-  ENABLE_DEPRECATED_API: false,
+  USE_DEPRECATED_API: false,
```
Comment (borismus) [inline src/main.js]: Slightly in favor of dropping this deprecated option too.

File: src/main.js (hunk 1 above)
Comment (gmarinov) [inline src/main.js]: +1

**Comment by borismus** [comment]
Please merge to master, and also include the newly added flags that disable the Cardboard UI.

File: src/main.js (hunk 1 above)
Comment (cvan) [inline src/main.js]: personally, I've spent way too much time chasing down regressions, intentional breakage without warning. I can remove, but I know it's causing _a lot_ of headache for folks today. and it's becoming a problem for some of us early adopters.

File: src/main.js (hunk 2 above)
Comment (cvan) [inline src/main.js]: I think that can be done in a separate issue/PR

Comment (borismus) [inline src/main.js]: SGTM

**Comment by borismus** [comment]
Could you merge to master please?

**Comment by borismus** [comment]
Closing due to inactivity.

**Comment by cvan** [comment]
that's fine - if folks share my concerns, it'll get reopened :)

**Pull request closed without merging**